app/Model/Place.php

```diff
@@ -45,9 +45,9 @@ public function sluggable()
      *
      * @return string
      */
-    public function getImagePathAttribute()
+    public function getImageUrlAttribute()
     {
-        return config("constant.path_upload_places").$this->image;
+        return asset(config("image.places.path_upload") . $this->image);
```

Comment:
please check image if not exist in `folder Path` . if not exist :  return url image default;